security-analysis/security-analysis-api/src/main/java/com/powsybl/security/condition/BranchThresholdCondition.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2025, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.security.condition;
+
+import com.powsybl.iidm.network.ThreeSides;
+
+/**
+ * Condition on a branch triggered by a threshold
+ *
+ * @author Bertrand Rix {@literal <bertrand.rix at artelys.com>}
+ */
+public class BranchThresholdCondition extends AbstractThresholdCondition {
+
+    public static final String NAME = "BRANCH_THRESHOLD_CONDITION";
+
+    // Side of the equipment on which to check the threshold
+    private final ThreeSides side;
```

Comment:
Branches only have 2 sides. `ThreeSides` is used for three-winding transformers which are not branches.
I think we should use another name for this class to avoid ambiguity...